ledger/participant-integration-api/BUILD.bazel

```diff
@@ -435,3 +435,36 @@ da_scala_benchmark_jmh(
         "//libs-scala/contextualized-logging",
     ],
 )
+
+da_scala_library(
+    name = "metering-verification-app-lib",
```

Comment:
What would speak against promoting this target to a dedicated library at the `//ledger/metering-verification` level? It is rather unusual for an app to be buried so deep in the library code

Reply:
Good idea.